src/layouts/slab.py

```diff
@@ -53,3 +51,68 @@
 SLAB_LAYOUT = cStruct("header" / SLAB_HEADER_LAYOUT, "nodes" / SLAB_NODE_LAYOUT[lambda this: this.header.bump_index])
 
 ORDER_BOOK_LAYOUT = cStruct(Padding(5), "account_flags" / ACCOUNT_FLAGS_LAYOUT, "slab_layout" / SLAB_LAYOUT, Padding(7))
+
+
+class Slab:
+    def __init__(self, header, nodes):
```

Comment:
Is it possible to add type hints here?

Reply:
This is not possible in this PR I think. Because I am using the raw return value from `construct` which is not typed. The end goal will be to create a wrapper class around these.

Reply:
I think we just need to explicitly define types for leaf node and inner node, and return `Optional[LeafNode, InnerNode]`.

Reply:
Discussed over Discord, agree that we will do it in a separate PR.